src/getColor.js

```diff
@@ -37,7 +37,7 @@ export default function getColor (color, options) {
 	// Object fixup
 	let space = color.space || color.spaceId;
 
-	if (!(space instanceof ColorSpace)) {
+	if (!ColorSpace.isColorSpace(space)) {
 		// Convert string id to color space object
 		color.space = ColorSpace.get(space);
```

Comment:
We could actually refactor this check to check if `space` is a string.

Reply:
the `util.isString()` is "slow" and came up as a hotspot when I was running the [texel](https://github.com/texel-org/color) benchmarks so I don't think we should add an additional call to `isString()`.

Reply:
Oh interesting. Maybe just `typeof space === "string"` then?

Reply:
Changed in 0123830e005e08e9511d8b8e63c6734dc7490e30